src/App.css

```diff
@@ -84,8 +84,9 @@ body {
 }
 
 .calSection {
+  margin-top: 1.5rem;
   display: flex;
-  gap: 1rem;
+  height: 100vh;
```

Comment:
- [ ] I appreciate your work on getting the layout to match the wireframe but if you noticed we have to scroll an empty space to see the footer. I went ahead and investigated a solution that you might like. we could just give a general style to our #root `height: 100vh` then set a height percentage for section for example `80%` and header `10%` footer `10%`. that would look neat for this given webiste